tests/tensorflow/test_compressed_graph.py

```diff
@@ -456,3 +457,45 @@ def test_quantize_outputs(desc: ModelDesc, _quantization_case_config):
 
     check_model_graph(compressed_model, desc.ref_graph_filename, _quantization_case_config.graph_dir,
                       desc.rename_resource_nodes)
+
+
+TYPE_HW = [(HWConfigType.CPU), (HWConfigType.GPU), (HWConfigType.VPU)]
+
+@pytest.fixture(scope='function', params=TYPE_HW)
+def hw_config_type(request):
+    type_hw = request.param
+    return type_hw
+
+
+TEST_HW_MODELS_DESC = [
+    ModelDesc('resnet50.pb', test_models.ResNet50, [1, 32, 32, 3]),
+    ModelDesc('inception_v3.pb', test_models.InceptionV3, [1, 75, 75, 3]),
+    ModelDesc('mobilenet_v2.pb', test_models.MobileNetV2, [1, 96, 96, 3]),
+]
+
+
+@pytest.mark.parametrize('desc', TEST_HW_MODELS_DESC, ids=[m.model_name for m in TEST_HW_MODELS_DESC])
+# pylint:disable=redefined-outer-name
```

Comment:
What do you think about using pytest parametrization, for example, @pytest.mark.parametrize('hw_config_type', TYPE_HW , ids=...) instead of the fixture?